tests/acceptance/features/webUITextEditor/editTextFiles.feature

```diff
@@ -22,3 +22,27 @@ Feature: textFiles
     And the user closes the text editor
     And the user opens file "New text file.txt" using the webUI
     Then line 1 of the text should be "other text before stuff"
+
+  @issue-core-36233
+  Scenario: Edit restored hidden text file
+    Given user "user1" has uploaded file with content "This is a hidden file" to "/.abc.txt"
+    And the user has enabled the setting to view hidden files on the webUI
+    When user "user1" deletes file "/.abc.txt" using the WebDAV API
+    And user "user1" restores the file with original path "/.abc.txt" using the trashbin API
+    And the user browses to the files page
+    Then as "user1" file "/.abc.txt" should exist
+    When user "user1" downloads file "/.abc.txt" using the WebDAV API
+    Then the downloaded content should be "This is a hidden file"
+#    When the user opens file "/.abc.txt" using the webUI
+#    Then line 1 of the text should be "This is a hidden file"
```

Comment:
Note: it is bit tricky to know what else to write here.
` When the user opens file "/.abc.txt" using the webUI` with the bug, the UI actually downloads the file (it does not recognize that it should open the text editor)
Unfortunately when the bug is fixed, this scenario will not start to fail. But probably it is too much effort to write special steps like `Then the text editor should not have opened`
So we just leave this as clues for the developer.